addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/gui/ParamMinerDialog.java

```diff
@@ -96,7 +116,37 @@ public void init(HttpMessage target) {
         this.pack();
     }
 
-    private void addUriSelectField(String fieldLabel, HttpMessage value, boolean editable) {
+    private void addWordlistSelectField(String fieldLabel, String filePath) {
+        text = new ZapTextField();
+        text.setText(filePath);
+        JButton browseButton = new JButton("browse");
+        JFileChooser fileChooser = new JFileChooser();
+        browseButton.addActionListener(
+                e -> {
+                    int returnVal = fileChooser.showOpenDialog(this);
+                    if (returnVal == JFileChooser.APPROVE_OPTION) {
```

Comment:
It isn't used any further so maybe don't bother with the intermediate store of the return value?

Ex: `if (fileChooser.showOpenDialog(this) == JFileChooser.APPROVE_OPTION) {`